packages/flame_tiled/example/lib/main.dart

```diff
@@ -38,4 +46,19 @@ class TiledGame extends FlameGame {
       );
     }
   }
+
+  @override
+  void update(double dt) {
+    super.update(dt);
+    time += dt;
+    final tiledMap = mapComponent.tileMap.map;
+    if (time % 10 < 5) {
+      camera.moveTo(Vector2(
```

Comment:
This (and the `else`) will create a lot of new unnecessary `Vector2` objects, you can create a vector outside of the update method that you use. (Since this will inspire how our users do this)